ramanujantools/pcf/pcf.py

```diff
@@ -77,7 +111,36 @@ def __eq__(self, other):
 
     def __repr__(self):
         return f"PCF({self.a_n}, {self.b_n})"
+    
+    def as_latex(self, depth: int = 3, start: int = 1, generic: bool = False, print_bool: bool = True):
```

Comment:
Add annotations that this function returns a str, and remove the `print_bool` functionality here as well